setup.py

```diff
@@ -22,16 +22,22 @@ def package_data(pkg, roots):
 
 setup(
     name='video-xblock',
-    version='0.1',
+    version='0.4.0',
     description='Video XBlock to embed videos hosted on different video platform into your courseware',
     license='GPL v3',
     packages=[
         'video_xblock',
     ],
+    dependency_links=[
+        # At the moment of writing PyPI hosts outdated version of xblock-utils, hence git
+        'git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils-1.0.2',
+    ],
     install_requires=[
-        'XBlock',
-        'xblock-utils',
-        'pycaption<1',  # The latest Python 2.7 compatible version
+        'XBlock>=0.4.14,<0.5.0',
+        'xblock-utils==1.0.2',
+        'pycaption<1.0',  # The latest Python 2.7 compatible version
+        'requests>=2.9.1,<3.0.0',
+        'babelfish>=0.5.5,<0.6.0',
```

Comment:
This is to be future-proof and prevent possible dependencies versions conflicts while safeguarding ourselves off major changes.

See [Specifying dependencies for a reusable component](https://blog.miguelgrinberg.com/post/the-package-dependency-blues) section.